src/test/java/org/apache/commons/lang3/builder/NoClassNameMultiLineToStringStyleTest.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.commons.lang3.builder;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.builder.ToStringStyleTest.Person;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Unit tests {@link org.apache.commons.lang3.builder.NoClassNameMultiLineToStringStyleTest}.
+ */
+public class NoClassNameMultiLineToStringStyleTest {
```

Comment:
As mentioned in #461 , maybe consider doing this work ahead of time?

@kinow , what do you think?

Reply:
Hi @Stzx 

Unfortunately I haven't had much experience with JUnit 5 yet. Looks to me like it will support multiple modifiers. But I couldn't find a recommendation, or even any deprecation, that would require dropping the `public` modifier. Do you know if that's a best practice, recommendation, or if it will be deprecated/required later?

I found at least one file in their repo using the `public` modifier: https://github.com/junit-team/junit5/blob/f52ded2d2f331ca7dba738b53afa5b1c0f22a85d/platform-tests/src/jmh/java/org/junit/jupiter/jmh/AssertionBenchmarks.java

Reply:
Found one comment that appears to be from someone involved with the project: https://stackoverflow.com/a/55230350

Doesn't seem to be required to use private, public, etc. It's up to the project.

Reply:
From the links you provided, as well as what I saw from the official examples ([junit5-jupiter-starter-maven](https://github.com/junit-team/junit5/blob/f52ded2d2f331ca7dba738b53afa5b1c0f22a85d/platform-tests/src/jmh/java/org/junit/jupiter/jmh/AssertionBenchmarks.java)) and [user guide](https://junit.org/junit5/docs/current/user-guide/#writing-tests) seems to be determined by the project?

Looks different styles of different projects.

Reply:
Yeah, and if it's not something that can be checked with checkstyle or with the other build tools, we have no clear way to enforce... and having to ask users to alter their PR's can be a bit frustrating for the contributors.

For this reason I think it might be easier to just keep things as they are, and accept new submissions using either public, private, or package-protected.